scripts/compare-llama-bench.py

```diff
@@ -114,7 +162,7 @@
 help_s = (
     "Columns to add to the table. "
     "Accepts a comma-separated list of values. "
-    f"Legal values: {', '.join(KEY_PROPERTIES[:-3])}. "
+    f"Legal values: {', '.join(LLAMA_BENCH_KEY_PROPERTIES[:-3])}. "
```

Comment:
```suggestion
    f"Legal values: {', '.join(LLAMA_BENCH_KEY_PROPERTIES[:-3])}. "
```

This needs to show legal values both for `llama-bench` and `test-backend-ops`.